com.cubrid.cubridmigration.core/src/com/cubrid/cubridmigration/cubrid/meta/CUBRIDSchemaFetcher.java

```diff
@@ -430,16 +427,37 @@ private Map<String, Table> buildCUBRIDTables(Connection conn, Catalog catalog, S
 		ResultSet rs = null;
 		Statement stmt = null;
 		try {
+			int dbVersion = getDBVersion(conn);
+			String sqlComment = 
+					dbVersion > COMMENT_SUPPORT_VERSION ? ", a.comment as column_comment, c.comment as table_comment" : "";
```

Comment:
isn't it, please check it.
`dbVersion > COMMENT_SUPPORT_VERSION`

Reply:
Yes, I will check.